$OpenBSD: patch-configure,v 1.2 2011/07/27 10:39:00 jasper Exp $

Let configure pickup the right bison; can be removed when updating
to a stable release.

--- configure.orig	Mon Jul 25 10:55:12 2011
+++ configure	Wed Jul 27 12:12:47 2011
@@ -4733,9 +4733,10 @@ fi
 done
 test -n "$YACC" || YACC="yacc"
 
-if test "$YACC" != "bison -y"; then
-	as_fn_error $? "bison is required" "$LINENO" 5
-fi
+#if test "$YACC" != "bison -y"; then
+#	as_fn_error $? "bison is required" "$LINENO" 5
+#fi
+YACC="bison -y"
 ac_ext=c
 ac_cpp='$CPP $CPPFLAGS'
 ac_compile='$CC -c $CFLAGS $CPPFLAGS conftest.$ac_ext >&5'
